code/game/machinery/alarm.dm

```diff
@@ -375,38 +375,6 @@
 		if(ATMOS_ALARM_DANGER)
 			icon_state = "alarm1"
 
-/obj/machinery/alarm/receive_signal(datum/signal/signal)

```

Comment:
For anyone wondering, this was used, back in the day for the atmospherics console.

It doesn't use signals anymore to register though so this is literally worthless now--it was also hanging all of game startup.